theme.json

```diff
@@ -76,7 +76,7 @@
 					"name": "40"
 				},
 				{
-					"size": "clamp(2.5rem, 8vw, 6.5rem))",
+					"size": "clamp(2.5rem, 8vw, 6.5rem)",
```

Comment:
Sizes `40` and `70` are using the calculation from [here](https://websemantics.uk/tools/responsive-font-calculator/), where as others not. Use the same calculation for all?

Reply:
> Use the same calculation for all?

Yes, we could do this. The main reason I didn't opt to use this for all of the spacing presets was because I was trying to stay close to the TT2 implementation - the ones that use the VW calculation are new sizes. It took a while to wrap up the initial spacing PR too 😅 But now that we have the initial version merged, it would be great to revisit with improvements!